contracts/01-tamagotchi/io/src/lib.rs

```diff
@@ -1,38 +1,43 @@
 #![no_std]
 
 use codec::{Decode, Encode};
-use gmeta::Metadata;
+use gmeta::{Metadata,In,InOut,Out};
```

Comment:
Please, use `cargo fmt` before PR.

Reply:
Got it.
Thank you. You are the first person who taught me coding step by step.